src/components/common/Dropdown/Dropdown.tsx

```diff
@@ -17,8 +16,8 @@ interface DropdownProps {
  * @흰색 테마:공통 대시보드에서 사용할 드롭다운, 앨범상세 곡별
  * @검은색 테마: 앨범 상세 트랙 별 정산액 추이 차트
  * @dropdownListData 드롭다운을 사용하는 컴포넌트에서 드롭다운 리스트에 넣어줄 데이터입니다. ex)["곡 명", "앨범 명"] 혹은 api 데이터
- * @theme 기본 테마는 흰색입니다 검은 테마를 사용하시려면 "black"을 입력해주세요
  * @hasSearchBar 드롭다운 리스트에 검색창이 필요하다면 이 속성을 true로 설정해주세요
+ * @theme 기본 테마는 흰색입니다 검은 테마를 사용하시려면 "black", 검색바와 같이 사용할 때는 "withSearchBar"를 사용하시면 됩니다
```

Comment:
param 키워드 사용도 좋을 것 같아요!
```suggestion
 * @param theme 기본 테마는 흰색입니다 검은 테마를 사용하시려면 "black", 검색바와 같이 사용할 때는 "withSearchBar"를 사용하시면 됩니다
```

Reply:
이거 변경할게요!